tests/testthat/indention_token_dependent/token_dependent_complex-out.R

```diff
@@ -2,30 +2,39 @@ call(call(
   2
 ))
 
-call(call(1,
-          2))
+call(call(
+  1,
+  2
+))
 # multi-line: no indention based on first vall
 call(a(b(c({
 }))))
 
+call(
+  call(
+    2
+  ),
+  5
+)
+
+
 call(call(
-  2),
-     5)
-
-
-call(call(1,
-          2, c(
-            3
-        )))
-
-call(1,
-     call2(3, 4, call(3,
-                      4, call(5, 6, call(
-                        2
-                      )
-                      )
-     )
-     )
+  1,
+  2, c(
```

Comment:
Right, we don't have such a rule yet. So if I understand you correctly, you think 
```r
call(
  2, 
  3, c(
    2, 
    3
))
```

Should be styled as 
```r
call(
  3, 
  3, 
  c(
    very_long_such, 
    that_we_cant_put_it_on_one_line
  )
)
```
If we can't put the whole sub-expression `c(...)` on one line?